sdcm/cluster_k8s/eks.py

```diff
@@ -261,6 +261,7 @@ def allowed_labels_on_scylla_node(self) -> list:
             ('app', 'static-local-volume-provisioner'),
             ('k8s-app', 'kube-proxy'),
             ('app.kubernetes.io/name', 'aws-ebs-csi-driver'),
+            ('app.kubernetes.io/component', 'chaos-daemon'),
```

Comment:
It must be conditioned:
```
if self.params.get('k8s_use_chaos_mesh'):
    allowed_labels_on_scylla_node.append(('app.kubernetes.io/component', 'chaos-daemon'))
```

The same in other places.

Reason: we can catch a bug in case it gets installed regardless of the config option.

Reply:
done